### PR TITLE
chore: change addon registry

### PR DESCRIPTION
**Pull request by haowen159**

### CODE REVIEW

**Review by nayutah — DISMISSED** [review]

File: addons/greptimedb/values.yaml [hunk 1]
```diff
@@ -12,8 +12,8 @@ clusterDomain: ".cluster.local"
 images:
   pullPolicy: IfNotPresent
   greptimedb:
-    registry: docker.io
-    repository: greptime/greptimedb
```
Comment (JashBook) [inline addons/greptimedb/values.yaml:16]: There is also a fixed registry
`docker.io/greptime/greptimedb-initializer:0.1.0-alpha.12`
https://github.com/apecloud/kubeblocks-addons/blob/chore/change_addon_registry/addons/greptimedb/templates/cmpd-datanode.yaml#L89

Multiple busybox missing registries and repositories
https://github.com/apecloud/kubeblocks-addons/blob/chore/change_addon_registry/addons/greptimedb/templates/cmpd-datanode.yaml#L58

File: addons/loki/values.yaml [hunk 2]
```diff
@@ -2,22 +2,23 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 images:
+  registry: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com
   # Overrides the image tag whose default is the chart appVersion.
   write:
     tag: 3.0.0
-    repository: docker.io/grafana/loki
```
Comment (JashBook) [inline addons/loki/values.yaml:8]: `kiwigrid/k8s-sidecar:1.24.3`
https://github.com/apecloud/kubeblocks-addons/blob/chore/change_addon_registry/addons/loki/templates/cmpd-backend.yaml#L38

File: addons/greatsql/values.yaml [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+  registry: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com
+  repository: apecloud/greatsql
   pullPolicy: IfNotPresent
   tag: 8.0.32-25
   prom:
-    repository: prom
+    repository: apecloud
     pullPolicy: IfNotPresent
     mysqld_exporter:
       name: mysqld-exporter
```
Comment (JashBook) [inline addons/greatsql/values.yaml:10]: missing registry
https://github.com/apecloud/kubeblocks-addons/blob/chore/change_addon_registry/addons/greatsql/templates/clusterdefinition.yaml#L49

File: addons/greptimedb/values.yaml [hunk 4]
```diff
@@ -15,6 +15,14 @@ images:
     registry: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com
     repository: apecloud/greptimedb
     tag: 0.3.2
+  initializer:
+    registry: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com
```
Comment (JashBook) [inline addons/greptimedb/values.yaml:19]: Use a unified parameter address registry to facilitate global settings.

**Review by ldming — DISMISSED** [review]

File: addons/etcd/opsdefinitions/defragmentation.yaml [hunk 5]
```diff
@@ -41,7 +41,7 @@ spec:
         podSpec:
           containers:
             - name: etcd-defragmentation
-              image: quay.io/coreos/etcd:v3.5.6
+              image: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/etcd:v3.5.6
```
Comment (weicao) [inline addons/etcd/opsdefinitions/defragmentation.yaml:44]: should not hard code image address in yaml

**Review by ldming — APPROVED** [review]

**Pull request merged**